site/app/templates/admin/RainbowCustomization.twig

```diff
@@ -155,6 +155,59 @@
             </div>
 
 
+            <div id="manual-grading" class="customization_item">
+                <h2>Manual Grading</h2>
+                <p class="rg_info_message">You may manually assign final grades to specific students. The note is only visible to the instructor, and it will only appear if instructor_notes is enabled under Display.</p>
+                <input type="hidden" name="csrf_token" value="{{ csrfToken }}" />
+                <input type="hidden" name="option" value="-1" />
+                <div class="option">
+                    <label for="manual_grading_user_id">
+                        <span>USER ID:</span>
+                        <input class="option-input" type="text" name="manual_grading_user_id" id="manual_grading_user_id">
+                    </label>
+<label for="manual_grading_grade">
+    <span>Final Grade:</span>
+    <select name="manual_grading_grade" id="manual_grading_grade">
+        <option disabled selected value> -- select an option -- </option>
+        {% set grades = ["A", "A-", "B+", "B", "B-", "C+", "C", "C-", "D+", "D", "F"] %}
+        {% for grade in grades %}
+            <option value="{{ grade }}">{{ grade }}</option>
+        {% endfor %}
+    </select>
+</label>
+
+                    <label for="manual_grading_note">
+                        <span>Note:</span>
+                        <input class="option-input note-input" type="text" name="manual_grading_note" id="manual_grading_note">
+                    </label>
+                    <button type="submit" class="btn btn-primary" onclick="addToTable(1)" aria-label="submit">
+                        <i class="fas fa-plus"></i>
+                    </button>
+                    {# This is a data table #}
+                    <table id="manual-grading-table" class="table table-striped table-bordered persist-area mobile-table">
+                        <thead>
+                            <tr style="text-align:left">
+                                <th>USER ID</th>
+                                <th>Final Grade</th>
+                                <th>Note</th>
+                                <th>Delete</th>
+                            </tr>
+                        </thead>
+                        <tbody id="manual-grading-table-body" style="text-align:left">
```

Comment:
I believe the `text-left` class can replace the inline CSS here.  I didn't test it, but I see this Bootstrap 3 class being used elsewhere in the codebase.

Reply:
Unfortunately, `class="text-left"` doesn't work since it is overridden by alignment in `class="table"`.

Reply:
Used an id selector to override the class selector. [b912515](https://github.com/Submitty/Submitty/pull/10653/commits/b912515195b09f448b35cd35f1734f9958150dec)